dlt/common/schema/typing.py

```diff
@@ -65,6 +64,7 @@
 ]
 """Known hints of a column used to declare hint regexes."""
 TWriteDisposition = Literal["skip", "append", "replace", "merge"]
+TLoaderMergeStrategy = Literal["scd2"]
```

Comment:
we should have a default strategy which is "delete-insert". and we'll add one more `merge` to support MERGE sql statements #1129

Reply:
Added in https://github.com/dlt-hub/dlt/pull/1168/commits/396ec59393f691573d68585befd99abcbb4fc5c8